packages/backend-defaults/src/entrypoints/database/connectors/postgres.test.ts

```diff
@@ -138,6 +140,114 @@ describe('postgres', () => {
       });
     });
 
+    it('uses the correct config when using azure', async () => {
```

Comment:
I'd expect test cases here for the client secret and managed identity credential configurations as well